Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM openjdk:17-jdk-alpine
```

Comment:
Is the lsc code really compatible with Java17?

Reply:
IMO using a JRE 17 with older code should be allright,